drivers/display/ssd1327.c

```diff
@@ -35,16 +43,22 @@ struct ssd1327_config {
 	uint8_t multiplex_ratio;
 	uint8_t prechargep;
 	uint8_t remap_value;
+	uint8_t phase_length;
+	uint8_t function_selection_b;
+	uint8_t precharge_voltage;
+	uint8_t vcomh_voltage;
 	bool color_inversion;
+	uint8_t *conversion_buf;
+	size_t conversion_buf_size;
 };
 
 struct ssd1327_data {
 	uint8_t contrast;
 	uint8_t scan_mode;
 };
 
-static inline int ssd1327_write_bus_cmd(const struct device *dev, const uint8_t cmd,
-					const uint8_t *data, size_t len)
+static inline int ssd1327_write_bus_cmd_mipi(const struct device *dev, const uint8_t cmd,
+					     const uint8_t *data, size_t len)
```

Comment:
How is this change related to "use the new L8 display format.". 

I think reading the documentation is a good starting point for you.
https://docs.zephyrproject.org/latest/contribute/contributor_expectations.html#pr-requirements
https://docs.zephyrproject.org/latest/contribute/style/code.html
Then some more work is needed to get this PR into a reviewable state.

Reply:
it also adds i2c.

Reply:
Is the wording ok now?